doc/ping.xml

```diff
@@ -24,7 +24,7 @@ xml:id="man.ping">
     <cmdsynopsis sepchar=" ">
       <command>ping</command>
       <arg choice="opt" rep="norepeat">
-        <option>-aAbBdDfhLnOqrRUvV46</option>
+        <option>-aAbBdDefhLnOqrRUvV46</option>
```

Comment:
I also wondered, if `-e` is the best option to be used. Looking that [`fping`](https://fping.org/) uses `-e` to `show elapsed time on return packets`:
```
$ fping ::1 -e
::1 is alive (0.042 ms)
```
which is really `fping` specific, we can use `-e` for this purpose.

Reply:
I'm down to whatever name you'd like. I've chosen `-e` because `-i` seemed too much for this option, `-d` was already used and the next letter in word `identification` is `e` :)

Reply:
I'll keep  `-e` unless anybody has a reason why not use it (if anybody reviews it).